src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/security/clients/MontaguIndirectClient.kt

```diff
@@ -9,7 +9,7 @@ import org.vaccineimpact.orderlyweb.db.AppConfig
 import org.vaccineimpact.orderlyweb.db.OrderlyUserRepository
 import org.vaccineimpact.orderlyweb.models.ErrorInfo
 import org.vaccineimpact.orderlyweb.security.authentication.MontaguAuthenticator
-import org.vaccineimpact.orderlyweb.security.authentication.khttpMontaguAPIClient
+import org.vaccineimpact.orderlyweb.security.providers.khttpMontaguAPIClient
```

Comment:
I moved this Montagu API Client and the new helper class for checking stuff with the Github API into a 'providers' folder.